.gitignore

```diff
@@ -103,4 +103,4 @@ tmp
 .ssh/
 
 #this is used by Jest-mongodb for testing only -
-globalConfig.json
\ No newline at end of file
+globalConfig.json.DS_Store
```

Comment:
please edit the .gitignore file and put a new-line between globalConfig.json and .DS_Store 
And then put a new line after the .DS_Store